tests/integration/test_binary_op.py

```diff
@@ -14,71 +14,67 @@
 #
 
 import numpy as np
+import pytest
 
 import cunumeric as num
 
+numpyX = np.array([1, 2, 3])
+numpyY = np.array([4, 5, 6])
 
-def test():
-    numpyX = np.array([1, 2, 3])
-    numpyY = np.array([4, 5, 6])
+x = pytest.fixture(lambda: num.array(numpyX))
+y = pytest.fixture(lambda: num.array(numpyY))
 
-    x = num.array(numpyX)
-    y = num.array(numpyY)
 
+def test_add(x, y):
     z = x + y
-    # # print(z)
     assert np.array_equal(z, numpyX + numpyY)
 
     z = x + 2
-    # print(z)
```

Comment:
I tried to use judgment in removing some print statements, since `pytest` will already spit out values when there failures.